app/assets/stylesheets/views/_overrides.scss

```diff
@@ -0,0 +1,8 @@
+/* Overrides styling of legends in finder-frontend.scss */
+legend.gem-c-title--margin-bottom-5 {
+  margin-bottom: $gutter-half !important;
+
+  @include media(tablet) {
+    margin-bottom: $gutter * 1.5 !important;
```

Comment:
Should `!important` be used? Was there no other solution?

Reply:
We shouldn't be using !important... 👎

Reply:
One for @tombye